exchanges/oex/oex_wrapper.go

```diff
@@ -0,0 +1,384 @@
+func (o *Oex) GetWebsocket() (*wshandler.Websocket, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (o *Oex) GetActiveOrders(getOrdersRequest *exchange.GetOrdersRequest) ([]exchange.OrderDetail, error) {
+	var resp []exchange.OrderDetail
+	var tempResp exchange.OrderDetail
+	tempData, err := o.getAllOpenOrderID()
+	if err != nil {
+		return resp, err
+	}
+	if len(getOrdersRequest.Currencies) == 0 {
+		for pair := range tempData {
+			getOrdersRequest.Currencies = append(getOrdersRequest.Currencies, currency.NewPairFromString(pair))
+		}
+		for x := range getOrdersRequest.Currencies {
+			key := exchange.FormatExchangeCurrency(o.Name, getOrdersRequest.Currencies[x]).String()
+			for tempKey, val := range tempData {
+				if tempKey != key {
+					continue
+				}
+				for y := 0; y < len(val); y++ {
+					tempData2, err := o.FetchOrderInfo(strconv.FormatInt(tempData[key][val[y]], 10), key)
+					if err != nil {
+						return resp, err
+					}
+					tempResp.Exchange = o.Name
+					tempResp.CurrencyPair = currency.NewPairFromString(key)
+					if strings.EqualFold(tempData2.Data.OrderInfo.Side, "BUY") {
+						tempResp.OrderSide = exchange.BuyOrderSide
+					} else {
+						tempResp.OrderSide = exchange.SellOrderSide
+					}
+					tempResp.Price = tempData2.Data.OrderInfo.Price
+					tempResp.Amount = tempData2.Data.OrderInfo.Volume
+					tempTime, err2 := strconv.ParseInt(tempData2.Data.OrderInfo.CreatedAt, 10, 64)
+					if err2 != nil {
+						return resp, err2
+					}
+					tempResp.OrderDate = time.Unix(tempTime, 9)
+					tempResp.ID = strconv.FormatInt(val[y], 10)
+					tempResp.ExecutedAmount = tempData2.Data.OrderInfo.DealVolume
+					tempResp.RemainingAmount = tempData2.Data.OrderInfo.Volume - tempData2.Data.OrderInfo.DealVolume
+					tempResp.Fee = tempData2.Data.OrderInfo.Fee
+					if getOrdersRequest.OrderSide == exchange.AnyOrderSide {
+						resp = append(resp, tempResp)
+						continue
+					}
+					if strings.EqualFold(getOrdersRequest.OrderSide.ToString(), tempResp.OrderSide.ToString()) {
+						resp = append(resp, tempResp)
+					}
+				}
+			}
+		}
+	}
+	return resp, nil
+}
+
+// GetOrderHistory retrieves account order information
+// Can Limit response to specific order status
+func (o *Oex) GetOrderHistory(getOrdersRequest *exchange.GetOrdersRequest) ([]exchange.OrderDetail, error) {
+	return nil, common.ErrNotYetImplemented
+}
+
+// GetFeeByType returns an estimate of fee based on the type of transaction
+func (o *Oex) GetFeeByType(feeBuilder *exchange.FeeBuilder) (float64, error) {
+	return 0, common.ErrFunctionNotSupported
+}
+
+// SubscribeToWebsocketChannels appends to ChannelsToSubscribe
+// which lets websocket.manageSubscriptions handle subscribing
+func (o *Oex) SubscribeToWebsocketChannels(channels []wshandler.WebsocketChannelSubscription) error {
+	o.Websocket.SubscribeToChannels(channels)
```

Comment:
Websocket is supported by this exchange, but isn't yet implemented and thus there's no need to call SubscribeToChannels so you can just return common.ErrNotYetImplemented for this and other websocket wrappers